data/ecosystem.yml

```
@@ -5,6 +5,7 @@ packages:
       '@shopify/stylelint-plugin',
       {
         install-command: 'yarn',
+        list-installed-versions-command: 'yarn list stylelint',
```

[suggest] Should we use `--pattern` according to the [Yarn doc](https://classic.yarnpkg.com/lang/en/docs/cli/list/)?

```suggestion
        list-installed-versions-command: 'yarn list --pattern stylelint',
```

---

Good catch, thank you